src/dbnode/storage/namespace_test.go

```diff
@@ -1466,13 +1461,13 @@ func TestNamespaceAggregateTiles(t *testing.T) {
 	targetShard0.EXPECT().
 		AggregateTiles(
 			sourceNsIDMatcher, targetNs, shard0ID, gomock.Len(2), gomock.Any(),
-			sourceBlockVolumes0, opts).
+			sourceBlockVolumes0, gomock.Any(), opts).
```

Comment:
I think it would be useful to pass a mock `onColdFlushNs` value in this test and assert that `Done()` gets called.